src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -78,7 +78,7 @@ std::string jsonRequestTreat
     if ((answer = parseDataP->ent.res.check(ciP, EntitiesRequest)) != "OK")
     {
       OrionError error(SccBadRequest, answer);
-      return error.render(ciP, "");
+      return setStatusCodeAndSmartRender(ciP, error);
```

Comment:
It's not very important. Do it if you want to.
The advantage of having it as a method it that it is easier to find. When you look at OrionError you see it.